File: last_word/last_word.go
package main

import (
	"strings"
)

func lengthOfLastWord(s string) int {
	words := strings.Fields(s)
	last := words[len(words)-1]

	return len(last)
}

func main() {
	s := "   fly me   to   the moon  "

	r := lengthOfLastWord(s)

	println(r)
}

File: last_word/last_word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        words = list(filter(None, s.split(" ")))

        return len(words[len(words) - 1])


def main():
    s = "   fly me   to   the moon  "

    sol = Solution()
    print(sol.lengthOfLastWord(s))


if __name__ == "__main__":
    main()

